src/library-authoring/create-container/CreateContainerModal.test.tsx

```diff
@@ -39,69 +24,71 @@ jest.mock('../routes', () => ({
   useLibraryRoutes: () => mockUseLibraryRoutes(),
 }));
 
-// Mock the useLibraryContext hook
-jest.mock('../common/context/LibraryContext', () => ({
-  useLibraryContext: jest.fn(),
-}));
-
-const defaultLibraryContext = {
-  libraryId: 'test-library',
-  collectionId: 'test-collection',
-  containerId: 'lct:org:lib:unit:test-container',
-  setContainerId: jest.fn(),
+const mockLibraryData = {
+  id: 'test-library',
+  title: 'Test Library',
+  description: 'A test library',
+  version: 1,
+  type: 'complex',
+  license: '',
+  canEditLibrary: true,
+  hasUnpublishedChanges: false,
+  hasUnpublishedDeletes: false,
+  numBlocks: 0,
+  lastPublished: null,
+  created: '2024-01-01T00:00:00Z',
+  updated: '2024-01-01T00:00:00Z',
+  publishedBy: null,
+  lastDraftCreated: null,
+  lastDraftCreatedBy: null,
   readOnly: false,
-  isLoadingLibraryData: false,
-  showOnlyPublished: false,
-  extraFilter: [],
-  isCreateCollectionModalOpen: false,
-  openCreateCollectionModal: jest.fn(),
-  closeCreateCollectionModal: jest.fn(),
-  createContainerModalType: ContainerType.Chapter,
-  setCreateContainerModalType: jest.fn(),
-  componentBeingEdited: undefined,
-  openComponentEditor: jest.fn(),
-  closeComponentEditor: jest.fn(),
-  setCollectionId: jest.fn(),
-};
-
-const defaultToastContext: ToastContextData = {
-  showToast: mockShowToast,
-  closeToast: mockCloseToast,
-  toastMessage: null,
-  toastAction: undefined,
+  allowLti: false,
+  allowPublicLearning: false,
+  allowPublicRead: false,
+  libraryBlockTypes: [],
+  learningPackage: null,
 };
 
-const renderComponent = (
-  toastContextValue = defaultToastContext,
-) => {
-  // Set up the mock implementation for useLibraryContext
-  jest.mocked(useLibraryContext).mockReturnValue(defaultLibraryContext);
+const mockBlockTypesData = [
+  { blockType: 'html', displayName: 'Text' },
+  { blockType: 'problem', displayName: 'Problem' },
+  { blockType: 'video', displayName: 'Video' },
+];
+
+const renderWithLibraryProvider = (libraryId: string, options: {
+  collectionId?: string;
+  containerId?: string;
+  path?: string;
+  params?: Record<string, string>;
+} = {}) => {
+  const {
+    collectionId, containerId, path, params,
+  } = options;
+
+  axiosMock.onGet(`/api/libraries/v2/${libraryId}/`).reply(200, mockLibraryData);
```

Comment:
@DanielVZ96 You can use [mockContentLibrary](https://github.com/openedx/frontend-app-authoring/blob/b5fd147eec9bb0095260f152a3193445c112ae72/src/library-authoring/data/api.mocks.ts#L41C23-L41C41) to mock this call and data